Mathlib/Order/ConditionallyCompleteLattice/Basic.lean

```diff
@@ -961,6 +979,52 @@ theorem exists_lt_of_ciInf_lt [Nonempty ι] {f : ι → α} (h : iInf f < a) : 
   @exists_lt_of_lt_ciSup αᵒᵈ _ _ _ _ _ h
 #align exists_lt_of_cinfi_lt exists_lt_of_ciInf_lt
 
+theorem csSup_of_not_bddAbove {s : Set α} (hs : ¬BddAbove s) : sSup s = sSup univ :=
+  ConditionallyCompleteLinearOrder.csSup_of_not_bddAbove s hs
+
+theorem csInf_of_not_bddBelow {s : Set α} (hs : ¬BddBelow s) : sInf s = sInf univ :=
+  ConditionallyCompleteLinearOrder.csInf_of_not_bddBelow s hs
```

Comment:
Are there any other places in mathlib where these two lemmas can be used to simplify a proof?

Reply:
Yes, even to simplify statements. Coming later.